editor/store/selectors.js

```diff
@@ -486,6 +486,16 @@ export const getBlocks = createSelector(
 	]
 );
 
+export const getBlocksByUid = createSelector(
+	( state, uids ) => {
+		return map( uids, ( uid ) => getBlock( state, uid ) );
+	},
+	( state ) => [
+		state.editor.present.blockOrder,
```

Comment:
How does this depend on `blockOrder` ?

Reply:
Also, since this depends on `getBlock`, it should define all of the dependants of `getBlock`.

Aside: `rememo@3.0.0` adds a helper utility for this via `getBlock.getDependants` ([changelog](https://github.com/aduth/rememo/blob/master/CHANGELOG.md#v300-2018-02-17)).

Reply:
I avoided using `getDependants` because of two things:

 - I have to map over the uids, to call the `getDependants` of the `getBlock`
 - I thought it's simpler to just add the global block state as dependants instead of using specific uids since a selector returning multiple blocks. 
 - blockOrder is necessary because getBlock depends on blockOrder see `getBlockDependantsCacheBust`